{0} setUpClass (tempest.api.compute.volumes.test_attach_volume.AttachVolumeMultiAttachTest) ... SKIPPED: Neutron is not available
{2} setUpClass (tempest.api.volume.admin.test_encrypted_volumes_extend.EncryptedVolumesExtendAttachedTest) ... SKIPPED: EncryptedVolumesExtendAttachedTest skipped as Neutron is not available
{1} tempest.api.compute.volumes.test_volume_snapshots.VolumesSnapshotsTestJSON.test_volume_snapshot_create_get_list_delete [5.830896s] ... ok
{0} tempest.api.compute.volumes.test_volumes_list.VolumesTestJSON.test_volume_list [0.073645s] ... ok
{0} tempest.api.compute.volumes.test_volumes_list.VolumesTestJSON.test_volume_list_param_limit [0.073277s] ... ok
{0} tempest.api.compute.volumes.test_volumes_list.VolumesTestJSON.test_volume_list_param_offset_and_limit [0.139138s] ... ok
{0} tempest.api.compute.volumes.test_volumes_list.VolumesTestJSON.test_volume_list_with_detail_param_limit [0.071239s] ... ok
{0} tempest.api.compute.volumes.test_volumes_list.VolumesTestJSON.test_volume_list_with_detail_param_offset_and_limit [0.137429s] ... ok
{0} tempest.api.compute.volumes.test_volumes_list.VolumesTestJSON.test_volume_list_with_details [0.068445s] ... ok
{2} tempest.api.volume.admin.test_groups.GroupsTest.test_group_create_show_list_delete [7.187504s] ... ok
{1} tempest.api.compute.volumes.test_volumes_get.VolumesGetTestJSON.test_volume_create_get_delete [2.890393s] ... ok
{0} tempest.api.compute.volumes.test_volumes_negative.VolumesNegativeTest.test_create_volume_with_invalid_size [0.191243s] ... ok
{0} tempest.api.compute.volumes.test_volumes_negative.VolumesNegativeTest.test_create_volume_with_size_zero [0.016259s] ... ok
{0} tempest.api.compute.volumes.test_volumes_negative.VolumesNegativeTest.test_create_volume_without_passing_size [0.015002s] ... ok
{0} tempest.api.compute.volumes.test_volumes_negative.VolumesNegativeTest.test_delete_invalid_volume_id [0.046184s] ... ok
{0} tempest.api.compute.volumes.test_volumes_negative.VolumesNegativeTest.test_delete_volume_without_passing_volume_id [0.011924s] ... ok
{0} tempest.api.compute.volumes.test_volumes_negative.VolumesNegativeTest.test_get_volume_without_passing_volume_id [0.010856s] ... ok
{0} tempest.api.compute.volumes.test_volumes_negative.VolumesNegativeTest.test_volume_delete_nonexistent_volume_id [0.046207s] ... ok
{0} tempest.api.compute.volumes.test_volumes_negative.VolumesNegativeTest.test_volume_get_nonexistent_volume_id [0.047085s] ... ok
{2} tempest.api.volume.admin.test_groups.GroupsTest.test_group_update [10.563491s] ... ok
{0} tempest.api.volume.admin.test_backends_capabilities.BackendsCapabilitiesAdminTestsJSON.test_compare_volume_stats_values [1.076712s] ... ok
{0} tempest.api.volume.admin.test_backends_capabilities.BackendsCapabilitiesAdminTestsJSON.test_get_capabilities_backend [0.534861s] ... ok
{2} tempest.api.volume.admin.test_qos.QosSpecsTestJSON.test_associate_disassociate_qos [0.710368s] ... ok
{2} tempest.api.volume.admin.test_qos.QosSpecsTestJSON.test_create_delete_qos_with_back_end_consumer [0.180185s] ... ok
{2} tempest.api.volume.admin.test_qos.QosSpecsTestJSON.test_create_delete_qos_with_both_consumer [0.172005s] ... ok
{2} tempest.api.volume.admin.test_qos.QosSpecsTestJSON.test_create_delete_qos_with_front_end_consumer [0.170064s] ... ok
{2} tempest.api.volume.admin.test_qos.QosSpecsTestJSON.test_get_qos [0.024726s] ... ok
{2} tempest.api.volume.admin.test_qos.QosSpecsTestJSON.test_list_qos [0.026676s] ... ok
{2} tempest.api.volume.admin.test_qos.QosSpecsTestJSON.test_set_unset_qos_key [0.155045s] ... ok
{1} tempest.api.volume.admin.test_group_snapshots.GroupSnapshotsV319Test.test_reset_group_snapshot_status [9.622976s] ... ok
{2} tempest.api.volume.admin.test_volume_quota_classes.VolumeQuotaClassesTest.test_show_default_quota [0.434938s] ... ok
{1} tempest.api.volume.admin.test_group_type_specs.GroupTypeSpecsTest.test_group_type_specs_create_show_update_list_delete [0.599052s] ... ok
{2} tempest.api.volume.admin.test_volume_quota_classes.VolumeQuotaClassesTest.test_update_default_quota [1.837996s] ... ok
{0} tempest.api.volume.admin.test_group_snapshots.GroupSnapshotsTest.test_create_group_from_group_snapshot [13.786858s] ... ok
{2} tempest.api.volume.admin.test_volume_quotas.VolumeQuotasAdminTestJSON.test_delete_quota [1.189471s] ... ok
{2} tempest.api.volume.admin.test_volume_quotas.VolumeQuotasAdminTestJSON.test_list_default_quotas [0.132824s] ... ok
{2} tempest.api.volume.admin.test_volume_quotas.VolumeQuotasAdminTestJSON.test_list_quotas [0.562253s] ... ok
{2} tempest.api.volume.admin.test_volume_quotas.VolumeQuotasAdminTestJSON.test_quota_usage [4.234782s] ... ok
{1} tempest.api.volume.admin.test_groups.GroupsV314Test.test_create_group_from_group [10.279685s] ... ok
{2} tempest.api.volume.admin.test_volume_quotas.VolumeQuotasAdminTestJSON.test_quota_usage_after_volume_transfer [4.606724s] ... ok
{2} tempest.api.volume.admin.test_volume_quotas.VolumeQuotasAdminTestJSON.test_show_quota_usage [0.153127s] ... ok
{2} tempest.api.volume.admin.test_volume_quotas.VolumeQuotasAdminTestJSON.test_update_all_quota_resources_for_tenant [0.557551s] ... ok
{2} setUpClass (tempest.api.volume.admin.test_volume_retype.VolumeRetypeWithMigrationTest) ... SKIPPED: Cinder multi-backend feature disabled
{1} tempest.api.volume.admin.test_groups.GroupsV320Test.test_reset_group_status [3.233945s] ... ok
{0} tempest.api.volume.admin.test_group_snapshots.GroupSnapshotsTest.test_delete_group_snapshots_following_updated_volumes [17.415218s] ... ok
{2} tempest.api.volume.admin.test_volume_retype.VolumeRetypeWithoutMigrationTest.test_available_volume_retype [5.447132s] ... ok
{1} tempest.api.volume.admin.test_snapshots_actions.SnapshotsActionsTest.test_reset_snapshot_status [0.404798s] ... ok
{0} tempest.api.volume.admin.test_group_snapshots.GroupSnapshotsTest.test_group_snapshot_create_show_list_delete [9.017455s] ... ok
{1} tempest.api.volume.admin.test_snapshots_actions.SnapshotsActionsTest.test_snapshot_force_delete_when_snapshot_is_creating [3.596762s] ... ok
{0} tempest.api.volume.admin.test_group_types.GroupTypesTest.test_group_type_create_list_update_show_delete [0.302807s] ... ok
{0} tempest.api.volume.admin.test_group_types.GroupTypesTest.test_group_type_list_by_optional_params [0.217313s] ... ok
{0} setUpClass (tempest.api.volume.admin.test_multi_backend.VolumeMultiBackendTest) ... SKIPPED: Cinder multi-backend feature disabled
{0} setUpClass (tempest.api.volume.admin.test_snapshot_manage.SnapshotManageAdminTest) ... SKIPPED: SnapshotManageAdminTest skipped as Neutron is not available
{2} tempest.api.volume.admin.test_volume_snapshot_quotas_negative.VolumeSnapshotQuotasNegativeTestJSON.test_quota_volume_gigabytes_snapshots [0.516199s] ... ok
{2} tempest.api.volume.admin.test_volume_snapshot_quotas_negative.VolumeSnapshotQuotasNegativeTestJSON.test_quota_volume_snapshots [0.108871s] ... ok
{1} tempest.api.volume.admin.test_snapshots_actions.SnapshotsActionsTest.test_snapshot_force_delete_when_snapshot_is_deleting [3.508635s] ... ok
{1} tempest.api.volume.admin.test_snapshots_actions.SnapshotsActionsTest.test_snapshot_force_delete_when_snapshot_is_error [3.602364s] ... ok
{0} tempest.api.volume.admin.test_user_messages.UserMessagesTest.test_delete_message [2.270832s] ... ok
{1} tempest.api.volume.admin.test_snapshots_actions.SnapshotsActionsTest.test_snapshot_force_delete_when_snapshot_is_error_deleting [3.582551s] ... ok
{1} tempest.api.volume.admin.test_snapshots_actions.SnapshotsActionsTest.test_update_snapshot_status [0.228513s] ... ok
{0} tempest.api.volume.admin.test_user_messages.UserMessagesTest.test_list_show_messages [1.862673s] ... ok
{2} tempest.api.volume.admin.test_volume_type_access.VolumeTypesAccessTest.test_volume_type_access_list [0.461110s] ... ok
{0} tempest.api.volume.admin.test_volume_hosts.VolumeHostsAdminTestsJSON.test_list_hosts [0.256117s] ... ok
{0} tempest.api.volume.admin.test_volume_hosts.VolumeHostsAdminTestsJSON.test_show_host [0.073784s] ... ok
{0} setUpClass (tempest.api.volume.admin.test_volume_manage.VolumeManageAdminTest) ... SKIPPED: Manage volume tests are disabled
{2} tempest.api.volume.test_availability_zone.AvailabilityZoneTestJSON.test_get_availability_zone_list [0.253919s] ... ok
{1} tempest.api.volume.admin.test_volume_pools.VolumePoolsAdminTestsJSON.test_get_pools_without_details [0.704293s] ... ok
{2} setUpClass (tempest.api.volume.test_image_metadata.VolumesImageMetadata) [0.000000s] ... FAILED

Captured traceback:
~~~~~~~~~~~~~~~~~~~
    Traceback (most recent call last):

      File "/usr/local/lib/python3.10/dist-packages/tempest/test.py", line 185, in setUpClass
    raise value.with_traceback(trace)

      File "/usr/local/lib/python3.10/dist-packages/tempest/test.py", line 178, in setUpClass
    cls.resource_setup()

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/test_image_metadata.py", line 40, in resource_setup
    cls.volume = cls.create_volume(imageRef=CONF.compute.image_ref)

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/base.py", line 118, in create_volume
    image = self.images_client.show_image(kwargs['imageRef'])

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/services/image/v2/images_client.py", line 119, in show_image
    resp, body = self.get(url)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 342, in get
    return self.request('GET', url, extra_headers, headers,

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 762, in request
    self._error_checker(resp, resp_body)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 872, in _error_checker
    raise exceptions.NotFound(resp_body, resp=resp)

    tempest.lib.exceptions.NotFound: Object not found
Details: {'message': 'No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\n\n\n', 'code': '404 Not Found', 'title': 'Not Found'}

{0} tempest.api.volume.admin.test_volume_services.VolumesServicesTestJSON.test_get_service_by_host_name [0.028858s] ... ok
{0} tempest.api.volume.admin.test_volume_services.VolumesServicesTestJSON.test_get_service_by_service_and_host_name [0.024790s] ... ok
{0} tempest.api.volume.admin.test_volume_services.VolumesServicesTestJSON.test_get_service_by_service_binary_name [0.027361s] ... ok
{0} tempest.api.volume.admin.test_volume_services.VolumesServicesTestJSON.test_get_service_by_volume_host_name [3.940715s] ... ok
{0} tempest.api.volume.admin.test_volume_services.VolumesServicesTestJSON.test_list_services [0.025300s] ... ok
{1} tempest.api.volume.admin.test_volume_quotas_negative.VolumeQuotasNegativeTestJSON.test_quota_volume_gigabytes [0.485448s] ... ok
{1} tempest.api.volume.admin.test_volume_quotas_negative.VolumeQuotasNegativeTestJSON.test_quota_volumes [0.396563s] ... ok
{2} tempest.api.volume.test_volume_absolute_limits.AbsoluteLimitsTests.test_get_volume_absolute_limits [0.155285s] ... ok
{1} tempest.api.volume.admin.test_volume_quotas_negative.VolumeQuotasNegativeTestJSON.test_volume_extend_gigabytes_quota_deviation [0.372219s] ... ok
{0} tempest.api.volume.admin.test_volume_types.VolumeTypesTest.test_volume_type_create_get_delete [0.299912s] ... ok
{0} tempest.api.volume.admin.test_volume_types.VolumeTypesTest.test_volume_type_encryption_create_get_update_delete [0.423775s] ... ok
{0} tempest.api.volume.admin.test_volume_types.VolumeTypesTest.test_volume_type_list [0.041282s] ... ok
{0} tempest.api.volume.admin.test_volume_types.VolumeTypesTest.test_volume_type_update [0.398904s] ... ok
{1} tempest.api.volume.admin.test_volume_services_negative.VolumeServicesNegativeTest.test_disable_log_reason_with_no_reason [0.020303s] ... ok
{1} tempest.api.volume.admin.test_volume_services_negative.VolumeServicesNegativeTest.test_disable_service_with_invalid_binary [0.031081s] ... ok
{1} tempest.api.volume.admin.test_volume_services_negative.VolumeServicesNegativeTest.test_enable_service_with_invalid_host [0.021047s] ... ok
{1} tempest.api.volume.admin.test_volume_services_negative.VolumeServicesNegativeTest.test_freeze_host_with_invalid_host [0.020087s] ... ok
{1} tempest.api.volume.admin.test_volume_services_negative.VolumeServicesNegativeTest.test_thaw_host_with_invalid_host [0.020603s] ... ok
{0} tempest.api.volume.admin.test_volume_types_extra_specs.VolumeTypesExtraSpecsTest.test_volume_type_extra_spec_create_get_delete [0.128383s] ... ok
{0} tempest.api.volume.admin.test_volume_types_extra_specs.VolumeTypesExtraSpecsTest.test_volume_type_extra_specs_list [0.066489s] ... ok
{0} tempest.api.volume.admin.test_volume_types_extra_specs.VolumeTypesExtraSpecsTest.test_volume_type_extra_specs_update [0.193213s] ... ok
{1} tempest.api.volume.admin.test_volume_types_extra_specs_negative.ExtraSpecsNegativeTest.test_create_invalid_body [0.018761s] ... ok
{1} tempest.api.volume.admin.test_volume_types_extra_specs_negative.ExtraSpecsNegativeTest.test_create_none_body [0.018157s] ... ok
{1} tempest.api.volume.admin.test_volume_types_extra_specs_negative.ExtraSpecsNegativeTest.test_create_nonexistent_type_id [0.038199s] ... ok
{1} tempest.api.volume.admin.test_volume_types_extra_specs_negative.ExtraSpecsNegativeTest.test_delete_nonexistent_volume_type_id [0.020512s] ... ok
{1} tempest.api.volume.admin.test_volume_types_extra_specs_negative.ExtraSpecsNegativeTest.test_get_nonexistent_extra_spec_name [0.026961s] ... ok
{1} tempest.api.volume.admin.test_volume_types_extra_specs_negative.ExtraSpecsNegativeTest.test_get_nonexistent_volume_type_id [0.022846s] ... ok
{1} tempest.api.volume.admin.test_volume_types_extra_specs_negative.ExtraSpecsNegativeTest.test_list_nonexistent_volume_type_id [0.022257s] ... ok
{1} tempest.api.volume.admin.test_volume_types_extra_specs_negative.ExtraSpecsNegativeTest.test_update_multiple_extra_spec [0.017875s] ... ok
{1} tempest.api.volume.admin.test_volume_types_extra_specs_negative.ExtraSpecsNegativeTest.test_update_no_body [0.017582s] ... ok
{1} tempest.api.volume.admin.test_volume_types_extra_specs_negative.ExtraSpecsNegativeTest.test_update_none_extra_spec_id [0.034767s] ... ok
{1} tempest.api.volume.admin.test_volume_types_extra_specs_negative.ExtraSpecsNegativeTest.test_update_nonexistent_extra_spec_id [0.040758s] ... ok
{1} tempest.api.volume.admin.test_volume_types_extra_specs_negative.ExtraSpecsNegativeTest.test_update_nonexistent_type_id [0.041649s] ... ok
{2} tempest.api.volume.test_volume_delete_cascade.VolumesDeleteCascade.test_volume_delete_cascade [8.561261s] ... ok
{1} tempest.api.volume.admin.test_volume_types_negative.VolumeTypesNegativeTest.test_create_volume_type_encryption_nonexistent_type_id [0.233810s] ... ok
{1} tempest.api.volume.admin.test_volume_types_negative.VolumeTypesNegativeTest.test_create_with_empty_name [0.018062s] ... ok
{1} tempest.api.volume.admin.test_volume_types_negative.VolumeTypesNegativeTest.test_create_with_repeated_name [0.153200s] ... ok
{1} tempest.api.volume.admin.test_volume_types_negative.VolumeTypesNegativeTest.test_delete_nonexistent_type_id [0.020129s] ... ok
{1} tempest.api.volume.admin.test_volume_types_negative.VolumeTypesNegativeTest.test_get_nonexistent_type_id [0.019968s] ... ok
{1} setUpClass (tempest.api.volume.admin.test_volumes_actions.VolumesActionsTest) ... SKIPPED: VolumesActionsTest skipped as Neutron is not available
{2} tempest.api.volume.test_volume_delete_cascade.VolumesDeleteCascade.test_volume_from_snapshot_cascade_delete [8.848715s] ... ok
{1} tempest.api.volume.test_extensions.ExtensionsTestJSON.test_list_extensions [0.210648s] ... ok
{0} tempest.api.volume.admin.test_volumes_list.VolumesListAdminTestJSON.test_volume_list_param_tenant [3.256343s] ... ok
{2} tempest.api.volume.test_volume_metadata.VolumesMetadataTest.test_crud_volume_metadata [0.336160s] ... ok
{2} tempest.api.volume.test_volume_metadata.VolumesMetadataTest.test_update_show_volume_metadata_item [0.246499s] ... ok
{1} tempest.api.volume.test_snapshot_metadata.SnapshotMetadataTestJSON.test_crud_snapshot_metadata [0.282225s] ... ok
{1} tempest.api.volume.test_snapshot_metadata.SnapshotMetadataTestJSON.test_update_show_snapshot_metadata_item [0.241494s] ... ok
{0} tempest.api.volume.test_versions.VersionsTest.test_list_versions [0.014041s] ... ok
{0} tempest.api.volume.test_versions.VersionsTest.test_show_version [0.206339s] ... ok
{2} tempest.api.volume.test_volume_transfers.VolumesTransfersV355Test.test_create_get_list_accept_volume_transfer [5.751759s] ... ok
{0} tempest.api.volume.test_volume_transfers.VolumesTransfersTest.test_create_get_list_accept_volume_transfer [5.475832s] ... ok
{2} tempest.api.volume.test_volume_transfers.VolumesTransfersV355Test.test_create_list_delete_volume_transfer [3.093104s] ... ok
{1} tempest.api.volume.test_volume_transfers.VolumesTransfersV357Test.test_create_get_list_accept_volume_transfer [5.334211s] ... ok
{0} tempest.api.volume.test_volume_transfers.VolumesTransfersTest.test_create_list_delete_volume_transfer [3.900250s] ... ok
{1} tempest.api.volume.test_volume_transfers.VolumesTransfersV357Test.test_create_list_delete_volume_transfer [3.912382s] ... ok
{1} setUpClass (tempest.api.volume.test_volumes_actions.VolumesActionsTest) ... SKIPPED: VolumesActionsTest skipped as Neutron is not available
{1} setUpClass (tempest.api.volume.test_volumes_extend.VolumesExtendAttachedTest) ... SKIPPED: VolumesExtendAttachedTest skipped as Neutron is not available
{0} tempest.api.volume.test_volumes_clone.VolumesCloneTest.test_create_from_bootable_volume [0.182856s] ... FAILED

Captured traceback:
~~~~~~~~~~~~~~~~~~~
    Traceback (most recent call last):

      File "/usr/local/lib/python3.10/dist-packages/tempest/common/utils/__init__.py", line 65, in wrapper
    return f(*func_args, **func_kwargs)

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/test_volumes_clone.py", line 67, in test_create_from_bootable_volume
    src_vol = self.create_volume(imageRef=img_uuid)

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/base.py", line 118, in create_volume
    image = self.images_client.show_image(kwargs['imageRef'])

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/services/image/v2/images_client.py", line 119, in show_image
    resp, body = self.get(url)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 342, in get
    return self.request('GET', url, extra_headers, headers,

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 762, in request
    self._error_checker(resp, resp_body)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 872, in _error_checker
    raise exceptions.NotFound(resp_body, resp=resp)

    tempest.lib.exceptions.NotFound: Object not found
Details: {'message': 'No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\n\n\n', 'code': '404 Not Found', 'title': 'Not Found'}


Captured pythonlogging:
~~~~~~~~~~~~~~~~~~~~~~~
    2025-02-25 20:11:36,307 84302 INFO     [tempest.lib.common.rest_client] Request (VolumesCloneTest:test_create_from_bootable_volume): 404 GET http://[LB_IP_ADDRESS]:9292/v2/images/5e42eec9-e25c-4a12-b50f-7c3845d9c222 0.179s
2025-02-25 20:11:36,307 84302 DEBUG    [tempest.lib.common.rest_client] Request - Headers: {'Content-Type': 'application/json', 'Accept': 'application/json', 'X-Auth-Token': '<omitted>'}
        Body: None
    Response - Headers: {'content-length': '139', 'content-type': 'application/json', 'x-openstack-request-id': 'req-2c3bc29d-2c65-457b-b7a3-f8ba06bf1be7', 'date': 'Tue, 25 Feb 2025 20:11:36 GMT', 'connection': 'close', 'status': '404', 'content-location': 'http://[LB_IP_ADDRESS]:9292/v2/images/5e42eec9-e25c-4a12-b50f-7c3845d9c222'}
        Body: b'{"message": "No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\\n\\n\\n", "code": "404 Not Found", "title": "Not Found"}'

{2} tempest.api.volume.test_volumes_clone_negative.VolumesCloneNegativeTest.test_create_from_volume_decreasing_size [3.996454s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_create_volume_from_deactivated_image [0.577061s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_create_volume_from_image_with_decreasing_size [0.415836s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_create_volume_with_invalid_size [0.023273s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_create_volume_with_nonexistent_snapshot_id [0.027681s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_create_volume_with_nonexistent_source_volid [0.034581s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_create_volume_with_nonexistent_volume_type [0.031217s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_create_volume_with_size_negative [0.015913s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_create_volume_with_size_zero [0.017480s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_create_volume_without_passing_size [0.015711s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_delete_invalid_volume_id [0.028428s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_delete_volume_without_passing_volume_id [0.011363s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_detach_volumes_with_invalid_volume_id [0.029755s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_get_invalid_volume_id [0.027289s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_get_volume_without_passing_volume_id [0.011554s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_list_volumes_detail_with_invalid_status [0.051249s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_list_volumes_detail_with_nonexistent_name [0.051185s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_list_volumes_with_invalid_status [0.034379s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_list_volumes_with_nonexistent_name [0.034160s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_reserve_volume_with_negative_volume_status [0.142279s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_reserve_volume_with_nonexistent_volume_id [0.026498s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_unreserve_volume_with_nonexistent_volume_id [0.026306s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_update_volume_with_empty_volume_id [0.011403s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_update_volume_with_invalid_volume_id [0.029302s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_update_volume_with_nonexistent_volume_id [0.029941s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_volume_delete_nonexistent_volume_id [0.026959s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_volume_extend_with_None_size [0.016476s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_volume_extend_with_non_number_size [0.015355s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_volume_extend_with_nonexistent_volume_id [0.029329s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_volume_extend_with_size_smaller_than_original_size [0.016261s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_volume_extend_without_passing_volume_id [0.029100s] ... ok
{2} tempest.api.volume.test_volumes_negative.VolumesNegativeTest.test_volume_get_nonexistent_volume_id [0.026262s] ... ok
{0} tempest.api.volume.test_volumes_clone.VolumesCloneTest.test_create_from_volume [8.574432s] ... ok
{1} tempest.api.volume.test_volumes_get.VolumesGetTest.test_volume_create_get_update_delete [7.877107s] ... ok
{2} setUpClass (tempest.api.volume.test_volumes_snapshots.VolumesSnapshotTestJSON) ... SKIPPED: VolumesSnapshotTestJSON skipped as Neutron is not available
{0} tempest.api.volume.test_volumes_extend.VolumesExtendTest.test_volume_extend [0.171620s] ... FAILED

Captured traceback:
~~~~~~~~~~~~~~~~~~~
    Traceback (most recent call last):

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/test_volumes_extend.py", line 36, in test_volume_extend
    volume = self.create_volume(imageRef=self.image_ref)

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/base.py", line 118, in create_volume
    image = self.images_client.show_image(kwargs['imageRef'])

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/services/image/v2/images_client.py", line 119, in show_image
    resp, body = self.get(url)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 342, in get
    return self.request('GET', url, extra_headers, headers,

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 762, in request
    self._error_checker(resp, resp_body)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 872, in _error_checker
    raise exceptions.NotFound(resp_body, resp=resp)

    tempest.lib.exceptions.NotFound: Object not found
Details: {'message': 'No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\n\n\n', 'code': '404 Not Found', 'title': 'Not Found'}


Captured pythonlogging:
~~~~~~~~~~~~~~~~~~~~~~~
    2025-02-25 20:11:48,165 84302 INFO     [tempest.lib.common.rest_client] Request (VolumesExtendTest:test_volume_extend): 404 GET http://[LB_IP_ADDRESS]:9292/v2/images/5e42eec9-e25c-4a12-b50f-7c3845d9c222 0.169s
2025-02-25 20:11:48,166 84302 DEBUG    [tempest.lib.common.rest_client] Request - Headers: {'Content-Type': 'application/json', 'Accept': 'application/json', 'X-Auth-Token': '<omitted>'}
        Body: None
    Response - Headers: {'content-length': '139', 'content-type': 'application/json', 'x-openstack-request-id': 'req-42567839-9966-439b-80ab-3756d3259903', 'date': 'Tue, 25 Feb 2025 20:11:48 GMT', 'connection': 'close', 'status': '404', 'content-location': 'http://[LB_IP_ADDRESS]:9292/v2/images/5e42eec9-e25c-4a12-b50f-7c3845d9c222'}
        Body: b'{"message": "No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\\n\\n\\n", "code": "404 Not Found", "title": "Not Found"}'

{0} tempest.api.volume.test_volumes_extend.VolumesExtendTest.test_volume_extend_when_volume_has_snapshot [8.924077s] ... ok
{1} tempest.api.volume.test_volumes_get.VolumesGetTest.test_volume_create_get_update_delete_as_clone [12.438265s] ... ok
{1} tempest.api.volume.test_volumes_get.VolumesGetTest.test_volume_create_get_update_delete_from_image [0.022058s] ... FAILED

Captured traceback:
~~~~~~~~~~~~~~~~~~~
    Traceback (most recent call last):

      File "/usr/local/lib/python3.10/dist-packages/tempest/common/utils/__init__.py", line 65, in wrapper
    return f(*func_args, **func_kwargs)

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/test_volumes_get.py", line 132, in test_volume_create_get_update_delete_from_image
    image = self.images_client.show_image(CONF.compute.image_ref)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/services/image/v2/images_client.py", line 119, in show_image
    resp, body = self.get(url)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 342, in get
    return self.request('GET', url, extra_headers, headers,

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 762, in request
    self._error_checker(resp, resp_body)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 872, in _error_checker
    raise exceptions.NotFound(resp_body, resp=resp)

    tempest.lib.exceptions.NotFound: Object not found
Details: {'message': 'No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\n\n\n', 'code': '404 Not Found', 'title': 'Not Found'}


Captured pythonlogging:
~~~~~~~~~~~~~~~~~~~~~~~
    2025-02-25 20:11:57,420 84304 INFO     [tempest.lib.common.rest_client] Request (VolumesGetTest:test_volume_create_get_update_delete_from_image): 404 GET http://[LB_IP_ADDRESS]:9292/v2/images/5e42eec9-e25c-4a12-b50f-7c3845d9c222 0.019s
2025-02-25 20:11:57,421 84304 DEBUG    [tempest.lib.common.rest_client] Request - Headers: {'Content-Type': 'application/json', 'Accept': 'application/json', 'X-Auth-Token': '<omitted>'}
        Body: None
    Response - Headers: {'content-length': '139', 'content-type': 'application/json', 'x-openstack-request-id': 'req-b7d73176-1bdb-4e31-826d-0db7289f341a', 'date': 'Tue, 25 Feb 2025 20:11:57 GMT', 'connection': 'close', 'status': '404', 'content-location': 'http://[LB_IP_ADDRESS]:9292/v2/images/5e42eec9-e25c-4a12-b50f-7c3845d9c222'}
        Body: b'{"message": "No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\\n\\n\\n", "code": "404 Not Found", "title": "Not Found"}'

{2} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshot_list_param_limit [0.028263s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshot_list_param_limit_equals_infinite [0.060331s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshot_list_param_limit_equals_zero [0.025854s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshot_list_param_marker [0.069116s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshot_list_param_offset [0.028171s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshot_list_param_sort_created_at_asc [0.030048s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshot_list_param_sort_created_at_desc [0.029171s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshot_list_param_sort_id_asc [0.030854s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshot_list_param_sort_id_desc [0.028781s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshot_list_param_sort_name_asc [0.030288s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshot_list_param_sort_name_desc [0.028312s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshots_list_details_with_params [0.090788s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_list.VolumesSnapshotListTestJSON.test_snapshots_list_with_params [0.088355s] ... ok
{0} tempest.api.volume.test_volumes_get.VolumesSummaryTest.test_show_volume_summary [0.214814s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_negative.VolumesSnapshotNegativeTestJSON.test_create_snapshot_with_nonexistent_volume_id [0.189863s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_negative.VolumesSnapshotNegativeTestJSON.test_create_snapshot_without_passing_volume_id [0.031479s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_negative.VolumesSnapshotNegativeTestJSON.test_list_snapshot_invalid_param_limit [0.014188s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_negative.VolumesSnapshotNegativeTestJSON.test_list_snapshots_invalid_param_marker [0.024058s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_negative.VolumesSnapshotNegativeTestJSON.test_list_snapshots_invalid_param_sort [0.016196s] ... ok
{1} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volume_list [0.046014s] ... ok
{1} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volume_list_by_name [0.041681s] ... ok
{1} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volume_list_details_by_name [0.058532s] ... ok
{1} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volume_list_details_pagination [0.266042s] ... ok
{1} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volume_list_details_with_multiple_params [0.121650s] ... ok
{1} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volume_list_pagination [0.181954s] ... ok
{1} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volume_list_param_display_name_and_status [0.037649s] ... ok
{1} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volume_list_with_detail_param_display_name_and_status [0.057597s] ... ok
{1} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volume_list_with_detail_param_marker [0.070896s] ... ok
{1} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volume_list_with_detail_param_metadata [0.067774s] ... ok
{1} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volume_list_with_details [0.064197s] ... ok
{1} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volume_list_with_param_metadata [0.044845s] ... ok
{1} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volumes_list_by_availability_zone [0.081563s] ... ok
{1} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volumes_list_by_bootable [0.078705s] ... ok
{1} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volumes_list_by_status [0.078226s] ... ok
{1} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volumes_list_details_by_availability_zone [0.063501s] ... ok
{1} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volumes_list_details_by_bootable [0.063653s] ... ok
{1} tempest.api.volume.test_volumes_list.VolumesListTestJSON.test_volumes_list_details_by_status [0.064214s] ... ok
{2} tempest.api.volume.test_volumes_snapshots_negative.VolumesSnapshotNegativeTestJSON.test_volume_from_snapshot_decreasing_size [7.203406s] ... ok

==============================
Failed 4 tests - output below:
==============================

setUpClass (tempest.api.volume.test_image_metadata.VolumesImageMetadata)
------------------------------------------------------------------------

Captured traceback:
~~~~~~~~~~~~~~~~~~~
    Traceback (most recent call last):

      File "/usr/local/lib/python3.10/dist-packages/tempest/test.py", line 185, in setUpClass
    raise value.with_traceback(trace)

      File "/usr/local/lib/python3.10/dist-packages/tempest/test.py", line 178, in setUpClass
    cls.resource_setup()

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/test_image_metadata.py", line 40, in resource_setup
    cls.volume = cls.create_volume(imageRef=CONF.compute.image_ref)

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/base.py", line 118, in create_volume
    image = self.images_client.show_image(kwargs['imageRef'])

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/services/image/v2/images_client.py", line 119, in show_image
    resp, body = self.get(url)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 342, in get
    return self.request('GET', url, extra_headers, headers,

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 762, in request
    self._error_checker(resp, resp_body)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 872, in _error_checker
    raise exceptions.NotFound(resp_body, resp=resp)

    tempest.lib.exceptions.NotFound: Object not found
Details: {'message': 'No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\n\n\n', 'code': '404 Not Found', 'title': 'Not Found'}


tempest.api.volume.test_volumes_clone.VolumesCloneTest.test_create_from_bootable_volume[id-cbbcd7c6-5a6c-481a-97ac-ca55ab715d16,image]
--------------------------------------------------------------------------------------------------------------------------------------

Captured traceback:
~~~~~~~~~~~~~~~~~~~
    Traceback (most recent call last):

      File "/usr/local/lib/python3.10/dist-packages/tempest/common/utils/__init__.py", line 65, in wrapper
    return f(*func_args, **func_kwargs)

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/test_volumes_clone.py", line 67, in test_create_from_bootable_volume
    src_vol = self.create_volume(imageRef=img_uuid)

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/base.py", line 118, in create_volume
    image = self.images_client.show_image(kwargs['imageRef'])

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/services/image/v2/images_client.py", line 119, in show_image
    resp, body = self.get(url)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 342, in get
    return self.request('GET', url, extra_headers, headers,

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 762, in request
    self._error_checker(resp, resp_body)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 872, in _error_checker
    raise exceptions.NotFound(resp_body, resp=resp)

    tempest.lib.exceptions.NotFound: Object not found
Details: {'message': 'No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\n\n\n', 'code': '404 Not Found', 'title': 'Not Found'}


Captured pythonlogging:
~~~~~~~~~~~~~~~~~~~~~~~
    2025-02-25 20:11:36,307 84302 INFO     [tempest.lib.common.rest_client] Request (VolumesCloneTest:test_create_from_bootable_volume): 404 GET http://[LB_IP_ADDRESS]:9292/v2/images/5e42eec9-e25c-4a12-b50f-7c3845d9c222 0.179s
2025-02-25 20:11:36,307 84302 DEBUG    [tempest.lib.common.rest_client] Request - Headers: {'Content-Type': 'application/json', 'Accept': 'application/json', 'X-Auth-Token': '<omitted>'}
        Body: None
    Response - Headers: {'content-length': '139', 'content-type': 'application/json', 'x-openstack-request-id': 'req-2c3bc29d-2c65-457b-b7a3-f8ba06bf1be7', 'date': 'Tue, 25 Feb 2025 20:11:36 GMT', 'connection': 'close', 'status': '404', 'content-location': 'http://[LB_IP_ADDRESS]:9292/v2/images/5e42eec9-e25c-4a12-b50f-7c3845d9c222'}
        Body: b'{"message": "No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\\n\\n\\n", "code": "404 Not Found", "title": "Not Found"}'


tempest.api.volume.test_volumes_extend.VolumesExtendTest.test_volume_extend[id-9a36df71-a257-43a5-9555-dc7c88e66e0e]
--------------------------------------------------------------------------------------------------------------------

Captured traceback:
~~~~~~~~~~~~~~~~~~~
    Traceback (most recent call last):

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/test_volumes_extend.py", line 36, in test_volume_extend
    volume = self.create_volume(imageRef=self.image_ref)

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/base.py", line 118, in create_volume
    image = self.images_client.show_image(kwargs['imageRef'])

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/services/image/v2/images_client.py", line 119, in show_image
    resp, body = self.get(url)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 342, in get
    return self.request('GET', url, extra_headers, headers,

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 762, in request
    self._error_checker(resp, resp_body)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 872, in _error_checker
    raise exceptions.NotFound(resp_body, resp=resp)

    tempest.lib.exceptions.NotFound: Object not found
Details: {'message': 'No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\n\n\n', 'code': '404 Not Found', 'title': 'Not Found'}


Captured pythonlogging:
~~~~~~~~~~~~~~~~~~~~~~~
    2025-02-25 20:11:48,165 84302 INFO     [tempest.lib.common.rest_client] Request (VolumesExtendTest:test_volume_extend): 404 GET http://[LB_IP_ADDRESS]:9292/v2/images/5e42eec9-e25c-4a12-b50f-7c3845d9c222 0.169s
2025-02-25 20:11:48,166 84302 DEBUG    [tempest.lib.common.rest_client] Request - Headers: {'Content-Type': 'application/json', 'Accept': 'application/json', 'X-Auth-Token': '<omitted>'}
        Body: None
    Response - Headers: {'content-length': '139', 'content-type': 'application/json', 'x-openstack-request-id': 'req-42567839-9966-439b-80ab-3756d3259903', 'date': 'Tue, 25 Feb 2025 20:11:48 GMT', 'connection': 'close', 'status': '404', 'content-location': 'http://[LB_IP_ADDRESS]:9292/v2/images/5e42eec9-e25c-4a12-b50f-7c3845d9c222'}
        Body: b'{"message": "No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\\n\\n\\n", "code": "404 Not Found", "title": "Not Found"}'


tempest.api.volume.test_volumes_get.VolumesGetTest.test_volume_create_get_update_delete_from_image[id-54a01030-c7fc-447c-86ee-c1182beae638,image,smoke]
-------------------------------------------------------------------------------------------------------------------------------------------------------

Captured traceback:
~~~~~~~~~~~~~~~~~~~
    Traceback (most recent call last):

      File "/usr/local/lib/python3.10/dist-packages/tempest/common/utils/__init__.py", line 65, in wrapper
    return f(*func_args, **func_kwargs)

      File "/usr/local/lib/python3.10/dist-packages/tempest/api/volume/test_volumes_get.py", line 132, in test_volume_create_get_update_delete_from_image
    image = self.images_client.show_image(CONF.compute.image_ref)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/services/image/v2/images_client.py", line 119, in show_image
    resp, body = self.get(url)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 342, in get
    return self.request('GET', url, extra_headers, headers,

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 762, in request
    self._error_checker(resp, resp_body)

      File "/usr/local/lib/python3.10/dist-packages/tempest/lib/common/rest_client.py", line 872, in _error_checker
    raise exceptions.NotFound(resp_body, resp=resp)

    tempest.lib.exceptions.NotFound: Object not found
Details: {'message': 'No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\n\n\n', 'code': '404 Not Found', 'title': 'Not Found'}


Captured pythonlogging:
~~~~~~~~~~~~~~~~~~~~~~~
    2025-02-25 20:11:57,420 84304 INFO     [tempest.lib.common.rest_client] Request (VolumesGetTest:test_volume_create_get_update_delete_from_image): 404 GET http://[LB_IP_ADDRESS]:9292/v2/images/5e42eec9-e25c-4a12-b50f-7c3845d9c222 0.019s
2025-02-25 20:11:57,421 84304 DEBUG    [tempest.lib.common.rest_client] Request - Headers: {'Content-Type': 'application/json', 'Accept': 'application/json', 'X-Auth-Token': '<omitted>'}
        Body: None
    Response - Headers: {'content-length': '139', 'content-type': 'application/json', 'x-openstack-request-id': 'req-b7d73176-1bdb-4e31-826d-0db7289f341a', 'date': 'Tue, 25 Feb 2025 20:11:57 GMT', 'connection': 'close', 'status': '404', 'content-location': 'http://[LB_IP_ADDRESS]:9292/v2/images/5e42eec9-e25c-4a12-b50f-7c3845d9c222'}
        Body: b'{"message": "No image found with ID 5e42eec9-e25c-4a12-b50f-7c3845d9c222<br /><br />\\n\\n\\n", "code": "404 Not Found", "title": "Not Found"}'



======
Totals
======
Ran: 203 tests in 191.7870 sec.
 - Passed: 189
 - Skipped: 10
 - Expected Fail: 0
 - Unexpected Success: 0
 - Failed: 4
Sum of execute time for each test: 246.8942 sec.

==============
Worker Balance
==============
 - Worker 0 (51 tests) => 0:02:57.994156
 - Worker 1 (67 tests) => 0:03:05.052694
 - Worker 2 (85 tests) => 0:03:11.779020
